packages/paypal-commerce-integration/src/paypal-commerce-fastlane/paypal-commerce-fastlane-payment-strategy.spec.ts

```diff
@@ -479,6 +479,41 @@ describe('PayPalCommerceFastlanePaymentStrategy', () => {
             });
             expect(paypalCommerceFastlaneUtils.updateStorageSessionId).toHaveBeenCalledWith(true);
         });
+
+        it('do not create an order if there is an error  while receiving a payment order', async () => {
+            await strategy.initialize(initializationOptions);
+
+            const paypalFastlaneComponent = await paypalFastlane.FastlaneCardComponent({});
+
+            jest.spyOn(paypalFastlaneComponent, 'getPaymentToken').mockRejectedValue(
+                new Error('input data error'),
+            );
+
+            try {
+                await strategy.execute(executeOptions);
+            } catch (error) {
+                expect(error).toBeInstanceOf(Error);
+                expect(paypalCommerceRequestSender.createOrder).not.toHaveBeenCalled();
+            }
+        });
+
+        it('do not call getPaymentToken when paying with a vaulted instrument', async () => {
```

Comment:
This test can be removed

Reply:
Why?

Reply:
Because if `getPaymentToken` method will be moved into `preparePaymentPayload` then it is not necessary to check it for vaulted instrument flow.